sos/report/plugins/seagate_ses.py

```diff
@@ -0,0 +1,56 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class SeagateSES(Plugin, RedHatPlugin):
+
+    short_desc = 'JBOD status'
+    plugin_name = 'seagate_ses'
+    plugin_timeout = 600
+    profiles = ('system', 'storage', 'hardware',)
+    packages = ('fwdownloader_megaraid',)
+
+    def setup(self):
+        res = self.collect_cmd_output('fwdownloader -ses')
+
+        # Finding actual SES devices and ignoring 0th element
+        # as it does not contain any device information
+        op_lst = res['output'].split("SES Device")[1:]
```

Comment:
The CI tests are failing here:

```
Traceback (most recent call last):
  File "/tmp/cirrus-ci-build/sos/report/__init__.py", line 1062, in setup
    plug.setup()
  File "/tmp/cirrus-ci-build/sos/report/plugins/seagate_ses.py", line 25, in setup
    op_lst = res['output'].split("SES Device")[1:]
TypeError: a bytes-like object is required, not 'str'
```

What does the output of `fwdownloader -ses` look like?

Reply:
Done